util/sorc/mkgfsawps.fd/compile_mkgfsawps_wcoss.sh

```diff
@@ -3,7 +3,7 @@ LMOD_EXACT_MATCH=no
 source ../../../sorc/machine-setup.sh > /dev/null 2>&1
 cwd=`pwd`
 
-if [ "$target" = "wcoss_dell_p3" ] || [ "$target" = "wcoss_cray" ] || [ "$target" = "hera" ] ; then
+if [ "$target" = "wcoss_dell_p3" ] || [ "$target" = "wcoss_cray" ] || [ "$target" = "wcoss2" ] ; then
```

Comment:
Is removing Hera as a target going to break anything there?

Reply:
Added Hera back in.